cpu/arm7_common/periph/pm.c

```diff
@@ -8,6 +8,7 @@
 
 /**
  * @ingroup     cpu_arm7_common
+ * @ingroup     drivers_periph_pm
```

Comment:
>  I know this isn't significant for this PR, since C-files are not compiled by doxygen, 

Yes, but some completionists still want to use doxygen to orient themselves in the C files ;-)